frontend/public/components/utils/withDragDropContext.tsx

```diff
@@ -0,0 +1,6 @@
+import {DragDropContext} from 'react-dnd';
```

Comment:
I missed this on previous review, but JS/TS filenames should be kebab cased. Let's call this something like `drag-drop-context.tsx`.